openmdao/vectors/vector.py

```diff
@@ -213,6 +214,22 @@ def _clone(self, initialize_views=False):
             vec._initialize_views()
         return vec
 
+    def _copy_views(self):
+        """
+        Return a lightweight copy containing just the views.
+
+        Returns
+        -------
+        <Vector>
+            Vector containing _name, _typ, and _views.
+        """
+        vec_cp = self.__class__(self._name, self._kind, self._system, self._root_vector,
```

Comment:
do we really need to recreate the whole Vector here?  We just need to copy the views of the existing vector, right?  Calling __init__ on the Vector calls _initialize_data() and _initialize_views() which do quite a bit of stuff that seems unnecessary here.

Reply:
Made a few adjustments so that we just need to save a copy of the views dictionary, then made the suggested change.